swagger-def/internal.yaml

```
@@ -1600,6 +1605,11 @@ definitions:
         type: string
         format: uuid
         x-nullable: true
+      ppm_id:
+        type: string
+        format: uuid
+        example: c56a4180-65aa-42ec-a945-5fd21dec0538
+        x-nullable: true
```

Similar to Final Incentive, I've borrowed this from our database comments on the table.

```suggestion
      ppm_id:
        description: >
          The PPM Shipment ID to associate with the assigned certificate.
        type: string
        format: uuid
        example: c56a4180-65aa-42ec-a945-5fd21dec0538
        x-nullable: true
```